cegui/include/CEGUI/Window.h

```diff
@@ -515,7 +515,10 @@ class CEGUIEXPORT Window :
     \return
         String object holding the Window type.
     */
-    const String& getType(void) const;
+    inline const String& getType() const
```

Comment:
Why are you making everything inline here? To make these things work like in a header-only library for optimization and avoiding bridging through library linkage stuffs?

Reply:
Optimization too, but at first I thought about readability. Window.cpp has 4000+ lines and was cluttered by these trivial getters etc. So I saved a code reader some effort. And it will probably work a bit faster, so it is a double win)